src/shogun/lib/tapkee/tapkee_shogun.cpp

```diff
@@ -1,8 +1,4 @@
-/*
- * This program is free software; you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation; either version 3 of the License, or
- * (at your option) any later version.
+/* This software is distributed under BSD 3-clause license (see LICENSE file).
```

Comment:
Can we please have these type of license headers shogun wide? Rather than that long and ugly bulky text in every file?


@iglesias @lisitsyn @vigsterkr